app/uk/gov/hmrc/helptosavefrontend/views/introduction/apply.scala.html

```diff
@@ -26,24 +26,32 @@
     sidebarLinks = Some(sidebar()),
     bodyClasses = None) {
 
-    @navigation(request.uri)
+    @navigation(request.uri, "Apply page")
 
     <section class="section">
         <h2>5. @Messages("hts.introduction.apply.title")</h2>
         <p>@Messages("hts.introduction.apply.par-1")</p>
         <ul class="bullets">
-            <li>@Messages("hts.introduction.apply.ul-1-1") <a title="Navigate to @Messages("hts.introduction.eligibility.title")" href=@routes.IntroductionController.getEligibility()>@Messages("hts.introduction.apply.ul-1-1-2")</a></li>
+            <li>@Messages("hts.introduction.apply.ul-1-1") <a title="Navigate to @Messages("hts.introduction.eligibility.title")"
+                                                              href=@routes.IntroductionController.getEligibility()
+                                                              data-ga-event-category = "links"
+                                                              data-ga-event-action = "Eligible"
+                                                              data-ga-event-label= "Apply page"
+                                                              class="ga-track-event"
+                                                              >@Messages("hts.introduction.apply.ul-1-1-2")</a></li>
             <li>@Messages("hts.introduction.apply.ul-1-2-1") @Messages("hts.introduction.apply.ul-1-2-2") @Messages("hts.introduction.apply.ul-1-2-3")</li>
         </ul>
         <p>@Messages("hts.introduction.apply.par-2")</p>
         <p>@Messages("hts.introduction.apply.par-3")</p>
         <p>@Messages("hts.introduction.apply.par-4")</p>
         @form(routes.EligibilityCheckController.getCheckEligibility(), 'class -> "group") {
-            @submit('_buttonClass -> "button--get-started") {
+            @submit('_buttonClass -> "button--get-started"
+              //TODO: '_data-ga-event-category -> "links" 'data-ga-event-action = "Start Now" 'data-ga-event-label= "Apply page"
```

Comment:
not sure how to fix this ,  adding attributes did not work for `@submit` or `@form`